src/Uno.UI/UI/Xaml/Controls/WebView/WebView.Android.cs

```diff
@@ -134,7 +134,8 @@ private void CreateAndLaunchMailtoIntent(Android.Content.Context context, string
 			}
 
 			_wasLoadedFromString = true;
-			_webView.LoadData(text, "text/html; charset=utf-8", "utf-8");
+			//Note : _webView.LoadData does not work properly on Android 10 even when we encode to base64.
+			_webView.LoadDataWithBaseURL(null, text, "text/html; charset=utf-8", "utf-8", null);
```

Comment:
does this work correctly on older versions of Android?

Reply:
Tested on simulators from Android 8 to 11 and it's working well.
I will try on more older to be sure

Reply:
With a UI Test, we'll validate on 5 and 11

Reply:
It's already tested indeed, once we add testing for Android11, it'll be validated as well.